.example.autotab.yaml

```diff
@@ -1,23 +1,27 @@
-default_email: you@gmail.com
 chrome_binary_location: /Applications/Google Chrome.app/Contents/MacOS/Google Chrome
 
 autotab_api_key: ... # Go to https://autotab.com/dashboard to get your API key, or
 # run `autotab record` with this field blank and you will be prompted to log in to autotab
 
-credentials:
-  - name: Google
-    login_url: https://accounts.google.com/v3/signin # Optional, defaults to https://{domain}/login
-    domains:
-    - google.com
+# Optional, programmatically login to services using "Login with Google" authentication
+google_credentials:
+  - name: default
```

Comment:
is name field optional? If not maybe easier to leave out and key by email, and maybe have a default: True that can be set on one email?